wg-outreach/meeting-notes/2019-04-01.md

```diff
@@ -17,7 +17,9 @@ April 1, 2019
     * https://github.com/electron/electron/issues/17456 
     * run through of electron build for macOS (sofia), windows (JD), and linux (david)
         * Sofia on macOS: all told, took 5-6 hours including troubleshooting
-        * Sofia's notes from building electron on MacOS: https://hackmd.io/Ef3mj2xXTBKR9grGsSZHog?both 
+        * Sofia's notes from building electron on MacOS: https://hackmd.io/Ef3mj2xXTBKR9grGsSZHog?both
+        * JD on Windows 10: all told, took days including troubleshooting
+        * JD's notes from buildong electron on Windows 10: https://hackmd.io/s/rJ67Tjc5V
     * Mentorship
```

Comment:
👆 Let me know if there's some tweaking to the hackmd url that's needed
_(I noticed it's a bit different than the other one posted)_.